3D Chess/Assets/Scripts/GameState.cs

```diff
@@ -34,9 +36,9 @@ public void ResetGame()
         pieceSpawner.CreateNewBoard(realm);
     }
 
-    private void Awake()
+    private async void Awake()
     {
-        realm = Realm.GetInstance();
+        realm = await CreateRealmAsync();
```

Comment:
This should probably show some spinner while the board is loading.

Reply:
See ed34a5c.